app-dicts/aspell-es/aspell-es-1.11.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+ASPELL_LANG="Spanish"
+ASPELL_VERSION=6
+
+inherit aspell-dict-r1
+
+MY_P="${P/aspell/aspell6}-2"
+
+SRC_URI="mirror://gnu/aspell/dict/${ASPELL_SPELLANG}/${MY_P}.tar.bz2"
+
+LICENSE="GPL-2"
```

Comment:
Would you be able to verify whether this isn't `GPL-2+`? (i.e. with explicit permission somewhere in the sources or README saying 'or newer')

Reply:
Checked and in the sources it is written that it is GPL-2